apps/app/src/app/pages/(examples)/examples/authentication/components/form.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, signal } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { provideIcons } from '@ng-icons/core';
+import { lucideGithub, lucideLoader2 } from '@ng-icons/lucide';
+import { HlmButtonDirective } from '@spartan-ng/ui-button-helm';
+import { HlmIconComponent } from '@spartan-ng/ui-icon-helm';
+import { HlmInputDirective } from '@spartan-ng/ui-input-helm';
+import { HlmSpinnerComponent } from '@spartan-ng/ui-spinner-helm';
+
+@Component({
+	selector: 'auth-example-form',
+	standalone: true,
+	imports: [HlmButtonDirective, HlmIconComponent, HlmInputDirective, HlmSpinnerComponent, FormsModule],
+	host: {
+		class: 'block',
+	},
+	providers: [provideIcons({ lucideGithub, lucideLoader2 })],
+	template: `
+		<div class="mx-auto flex w-full flex-col justify-center space-y-6 sm:w-[350px]">
+			<div class="flex flex-col space-y-2 text-center">
+				<h1 class="text-2xl font-semibold tracking-tight">Create an account</h1>
+				<p class="text-muted-foreground text-sm">Enter your email below to create your account</p>
+			</div>
+			<div class="grid gap-6">
+				<form (ngSubmit)="send()">
+					<div class="grid gap-2">
+						<div class="grid gap-1">
+							<label
+								class="sr-only text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
+								for="email"
+							>
+								Email
+							</label>
+							<input
+								class="flex h-9 w-full focus-visible:ring-1"
+								hlmInput
+								placeholder="Email"
+								type="email"
+								id="email"
+								placeholder="name@example.com"
+								autocomplete="off"
+								type="email"
+							/>
+						</div>
+
+						<button
+							hlmBtn
+							[disabled]="isLoading()"
+							class="h-9 px-4 py-2 focus-visible:ring-1"
+							animate-spin
```

Comment:
animate-spin is normally a class that's added to the icon. I think we can remove it here